test/testsensor.c

```diff
@@ -111,7 +111,7 @@ int main(int argc, char **argv)
         bool done = false;
         SDL_Event event;
 
-        SDL_CreateWindow("Sensor Test", 0, 0, SDL_WINDOW_FULLSCREEN);
+        SDL_Window* window = SDL_CreateWindow("Sensor Test", 0, 0, SDL_WINDOW_FULLSCREEN);
```

Comment:
Without commenting on the aesthetics of this proposed change, it is able to warn us about this questionable code in the tests.

Reply:
Glad you see the value in the change. Will add the null check.